package.json

```diff
@@ -120,5 +120,8 @@
         "babel-module": {}
       }
     }
+  },
+  "dependencies": {
+    "normalize-url": "^1.9.0"
```

Comment:
This would assume that the uri always begins with a slash, which may not always be the case